mobile/src/components/common/button/button.tsx

```diff
@@ -1,7 +1,5 @@
 import React, { FC } from 'react';
-import { Pressable } from 'react-native';
-
-import { Text } from '~/components/common/common';
+import { Text, Pressable } from '~/components/common/common';
```

Comment:
We only re-export components that we use throughout the application, not in specific places. Return it back